clang-tools-extra/clang-doc/Serialize.cpp

```diff
@@ -710,11 +897,12 @@ emitInfo(const RecordDecl *D, const FullComment *FC, Location Loc,
 
     // What this is a specialization of.
     auto SpecOf = CTSD->getSpecializedTemplateOrPartial();
-    if (auto *CTD = dyn_cast<ClassTemplateDecl *>(SpecOf))
-      Specialization.SpecializationOf = getUSRForDecl(CTD);
-    else if (auto *CTPSD =
-                 dyn_cast<ClassTemplatePartialSpecializationDecl *>(SpecOf))
-      Specialization.SpecializationOf = getUSRForDecl(CTPSD);
+    if (auto *SpecPtr = dyn_cast<ClassTemplateDecl *>(SpecOf)) {
+      Specialization.SpecializationOf = getUSRForDecl(SpecPtr);
+    } else if (auto *SpecPtr =
+                   dyn_cast<ClassTemplatePartialSpecializationDecl *>(SpecOf)) {
+      Specialization.SpecializationOf = getUSRForDecl(SpecPtr);
+    }
```

Comment:
LLVM style omits curly braces for single-statement bodies: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
I think I've gotten them all.

Reply:
I'll mark this done, since clang-format w/ `RemoveBracesLLVM: true` fixed up this and other patches in the stack.